addons/web/static/tests/fields/basic_fields_tests.js

```diff
@@ -1902,6 +1902,44 @@ QUnit.module('basic_fields', {
         assert.verifySteps(['destroy']);
     });
 
+    QUnit.test('graph dashboard widget does not need nv to be fully loaded', function (assert) {
```

Comment:
Title is weird. Suggestion 'graph dashboard widget can be destroyed when nv is partially loaded'.